code/modules/projectiles/boxes_magazines/_box_magazine.dm

```diff
@@ -13,15 +13,27 @@
 	w_class = WEIGHT_CLASS_TINY
 	throw_speed = 3
 	throw_range = 7
+	///list containing the actual ammo within the magazine
 	var/list/stored_ammo = list()
+	///type that the magazine will be searching for, rejects if not a subtype of
 	var/ammo_type = /obj/item/ammo_casing
+	///maximum amount of ammo in the magazine
 	var/max_ammo = 7
-	var/multiple_sprites = 0
+	///Controls how sprites are updated for the ammo box

```

Comment:
//* for multi-line comments iirc

Reply:
@actioninja this works, but it's treated as markdown so all these line will run together

https://commonmark.org/help/